keepassxc-browser/background/page.js

```diff
@@ -221,3 +234,61 @@ page.removePageInformationFromNotExistingTabs = async function() {
         }
     }
 };
+
+// Retrieves the credentials. Returns cached values when found.
```

Comment:
Will this make it difficult to poll for TOTP when requested, versus when loaded?

Reply:
I still need to make the separate TOTP request.